### PR TITLE
Age

### PR DESCRIPTION
**Pull request by IliaKopylov**
Create agePositive.js, add to selectors, testData, expected. Change wdioConfig included runs.

### CODE REVIEW

File: test/extended/agePositive.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import sel from '../../data/selectors'; // импорт в переменную 'sel' константы из файла selectors.js
```
Comment (michael120288) [inline test/extended/agePositive.js:1]: Delete comments or at least do it on english

File: test/extended/agePositive.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import exp from '../../data/expected.json'
+import  {name, age, gender,story} from '../../data/testData';
+import inputValues4 from '../../helpers/methods'
+//const sel = require('../data/selectors'); старый вариант
```
Comment (michael120288) [inline test/extended/agePositive.js:5]: delete

File: test/extended/agePositive.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+        it('TC-069 Age field can accept "1"', function () {
+            $(sel.agePlaceHolder).setValue(age.min)
+            $(sel.pageClick).click();
+            let errorAlert = $(sel.alert).isDisplayed();
+            expect(errorAlert).toEqual(false);
+        });
+
+        it('TC-070 Age = 999999999999', function () {
+            //expect(submitButton).toEqual(true); //compare {title} (actual) and "MLH trial" (expected)
```
Comment (michael120288) [inline test/extended/agePositive.js:35]: you don't need to leave the comments that we don't use

File: test/extended/agePositive.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            $(sel.agePlaceHolder).setValue(age.default)
+            $(sel.pageClick).click();
+            let errorAlert = $(sel.alert).isDisplayed();
+            expect(errorAlert).toEqual(false);
+        });
+    });
+
+
+
+});
```
Comment (michael120288) [inline test/extended/agePositive.js:53]: add empty line in the end)

File: test/extended/agePositive.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        it('TC-070 Age = 999999999999', function () {
+            //expect(submitButton).toEqual(true); //compare {title} (actual) and "MLH trial" (expected)
+            $(sel.agePlaceHolder).setValue(age.max)
+            $(sel.pageClick).click();
+            let errorAlert = $(sel.alert).isDisplayed();
+            expect(errorAlert).toEqual(false);
+        });
+
+        it('TC - 071 Age field can accept digits "1234567890"', function () {
+            //expect(submitButton).toEqual(true); //compare {title} (actual) and "MLH trial" (expected)
```
Comment (michael120288) [inline test/extended/agePositive.js:43]: line that we don't use

File: test/extended/ageNegative.js [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+            if(errorAlert === true) {
+                let errorText = $(sel.alert).getText();
+                expect(errorText).toEqual(exp.ageError1);
+            } else {
+                expect(errorAlert).toEqual(true);
+            }
+    });
+
+
+
```
Comment (michael120288) [inline test/extended/ageNegative.js:170]: empty lines that need to delete

File: data/expected.json [hunk 7]
```diff
@@ -12,5 +12,9 @@
...
   "imageLabel": "5. Upload an image (optional)",
   "submitText": "Create!",
-  "errorMassage": "70 symbols max"
-}
+  "errorMassage": "70 symbols max",
+
+          "agePlaceHolder" : "Hero's age",
+          "ageError1": "looks like unreal age",
+          "ageError2": "Required"
+}
```
Comment (michael120288) [inline data/expected.json:20]: add an empty line in the end)

File: data/selectors.js [hunk 8]
```diff
@@ -14,7 +14,15 @@ const elements = {
     "submit":".ant-btn",
     "tryAgain": ".btn-primary",
     "google": ".gLFyf",
-    "errorMessage": ".ant-form-item-explain"
+    "errorMessage": ".ant-form-item-explain",
+
```
Comment (michael120288) [inline data/selectors.js:18]: delete line

**Review by michael120288 — CHANGES_REQUESTED** [review]
look at my comments

**Review by michael120288 — CHANGES_REQUESTED** [review]
again

**Review by IliaKopylov — COMMENTED** [review]
I fixed all

File: test/extended/ageNegative.js [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+            $(sel.pageClick).click();
+            let errorAlert = $(sel.alert).isDisplayed();
+            if(errorAlert === true) { // если ошибка появляется
+                let errorText = $(sel.alert).getText();   // берем текст ошибки
+                expect(errorText).toEqual(exp.ageError1); // проверка соответсвия текста ошибки
+            } else {
+                expect(errorAlert).toEqual(true); // проверка если ошибка не появилась
+            }
+        });
+*/
```
Comment (michael120288) [inline test/extended/ageNegative.js:84]: Delete

File: test/extended/ageNegative.js [hunk 10]
```diff
@@ -0,0 +1,190 @@
+import sel from '../../data/selectors'; // импорт в переменную 'sel' константы из файла selectors.js
+import exp from '../../data/expected.json';
+import  {name, age, gender,story} from '../../data/testData';
+import all from '../../helpers/methods';
+// import inputFieldClear from '../../helpers/methods';
+//const sel = require('../data/selectors'); старый вариант
```
Comment (michael120288) [inline test/extended/ageNegative.js:6]: delete

File: helpers/methods.js [hunk 11]
```diff
@@ -1,12 +1,21 @@
...
+    "inputFieldClear": function inputFieldClear(ageFieldValue) {
+        for (let i = 0; i < ageFieldValue.length; i++) {
+            browser.keys('Backspace');
+        }
+    },
 }
 
-module.exports = inputValues4;
\ No newline at end of file
+//export default all;
```
Comment (michael120288) [inline helpers/methods.js:20]: delete

File: test/extended/ageNegative.js [hunk 12]
```diff
@@ -0,0 +1,190 @@
+import sel from '../../data/selectors'; // импорт в переменную 'sel' константы из файла selectors.js
+import exp from '../../data/expected.json';
+import  {name, age, gender,story} from '../../data/testData';
+import all from '../../helpers/methods';
+// import inputFieldClear from '../../helpers/methods';
```
Comment (michael120288) [inline test/extended/ageNegative.js:5]: Delete

**Review by michael120288 — CHANGES_REQUESTED** [review]
again

**Comment by michael120288** [comment]
Again

**Pull request closed without merging**